### PR TITLE
Add support for multiple audio tracks

### PR DESCRIPTION
**Pull request by mividtim**
### Pivotal Tracker
Implements player-plugin side functionality in support of https://www.pivotaltracker.com/story/show/167762544

### Changes
- Add a `selectAudioTrack` method to the JS interface
- Wrap `BrightcoveAudioTracksController` in a class in the same package, to expose the protected `selectAudioTrack` method we need to change audio tracks

### CODE REVIEW

File: android/src/main/java/jp/manse/BrightcovePlayerManager.java [hunk 1]
```diff
@@ -108,7 +108,11 @@ public void receiveCommand(BrightcovePlayerView view, int commandType, @Nullable
         switch (commandType) {
             case COMMAND_SEEK_TO: {
                 view.seekTo((int)(args.getDouble(0) * 1000));
-                return;
+                break;
+            }
+            case COMMAND_SELECT_AUDIO_TRACK: {
+                view.selectAudioTrack(args.getString(0));
```
Comment (greenrift) [inline android/src/main/java/jp/manse/BrightcovePlayerManager.java]: @mividtim Should we check that a: the 0 index is not null (args may not be null, but index may be) and b: That it is actually a string?

**Review by greenrift — APPROVED** [review]
One comment on risk of NPE that might need to be adjusted otherwise looks good to me

File: android/src/main/java/jp/manse/BrightcovePlayerManager.java (hunk 1 above)
Comment (mividtim) [reply android/src/main/java/jp/manse/BrightcovePlayerManager.java]: I'm happy to add some type safety, but it's impossible to go too far with it. The interface for ReadableArray is pretty crappy. It's just an interface, so there are no `thrown`s defined for the methods. It's just fragile by design. You can't look ahead to see what the type is - you're just expected to send the right type. The RN bridge is terrible. If you toss a NaN into a Double, it'll crash the application!

Comment (mividtim) [reply android/src/main/java/jp/manse/BrightcovePlayerManager.java]: On the JS side, we have type safety defined on the CrewUtils methods. I think that's about the best we can do, unfortunately. I just added in args null- and size-checks.

**Pull request merged**